Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentStoreSettings.cs

```diff
@@ -130,16 +130,10 @@ public int AssumeAvailableReplicaCount
         /// </summary>
         public int ParallelCopyFilesLimit { get; set; } = DefaultParallelCopyFilesLimit;
 
-        private IReadOnlyList<TimeSpan> _retryIntervalForCopies = CacheCopierDefaultRetryIntervals;
-
         /// <summary>
         /// Delays for retries for file copies
         /// </summary>
-        public IReadOnlyList<TimeSpan> RetryIntervalForCopies
-        {
-            get => _retryIntervalForCopies ?? CacheCopierDefaultRetryIntervals;
-            set => _retryIntervalForCopies = value;
-        }
+        public IReadOnlyList<TimeSpan> RetryIntervalForCopies { get; set; } = CacheCopierDefaultRetryIntervals;

```

Comment:
Do we have to worry about the behavior chagne where in the old code if someone set it to null, we'd still return the defaults. Now we'll start returning nul...

Reply:
These are all set via settings, so this should not be an issue.

---
In reply to: [307925941](https://github.com/microsoft/BuildXL/pull/667#discussion_r307925941) [](ancestors = 307925941)